eqt/ui/UIFormWidget.py

```diff
@@ -68,6 +68,29 @@ def getWidget(self, name, role='field'):
             return self.widgets['{}_{}'.format(name, role)]
         raise ValueError('Unexpected role: expected any of {}, got {}'.format(allowed_roles, role))
 
+    def setWidgetVisible(self, name, visible):
```

Comment:
Do you not want to hide both field and label?

Reply:
that's what it's doing?